pkgs/os-specific/darwin/apple-source-releases/system_cmds/meson.build

```diff
@@ -0,0 +1,425 @@
+#         'gcore/vanilla.c',
+#         'gcore/vm.c'
+#     ]
+# )
+
+executable(
+    'getconf',
+    c_args : '-DAPPLE_GETCONF_UNDERSCORE',
+    include_directories : 'getconf',
+    install : true,
+    sources : [
+        'getconf/getconf.c',
+    ] + pgperf.process(
+        [
+            'getconf/confstr.gperf',
+            'getconf/limits.gperf',
+            'getconf/unsigned_limits.gperf',
+            'getconf/progenv.gperf',
+            'getconf/sysconf.gperf',
+            'getconf/pathconf.gperf'
+        ]
+    )
+)
+
+executable(
+    'getty',
+    install : true,
+    sources : [
+        'getty/chat.c',
+        'getty/init.c',
+        'getty/main.c',
+        'getty/subr.c'
+    ]
+)
+
+executable('hostinfo', install : true, sources : 'hostinfo/hostinfo.c')
+
+executable(
+    'iosim',
+    dependencies : [ core_foundation, iokit ],
+    include_directories : 'at',
+    install : true,
+    sources : 'iosim/iosim.c'
+)
+
+executable(
+    'iostat',
+    build_by_default : sdk_version.version_compare('>=11'),
+    dependencies : [ core_foundation, iokit ],
+    install : sdk_version.version_compare('>=11'),
+    sources : 'iostat/iostat.c'
+)
+
+# Requires xnu headers
+# executable('kpgo', install : true, sources : 'kpgo/kpgo.c')
+
+executable(
+    'latency',
+    build_by_default : sdk_version.version_compare('>=12'),
+    dependencies : ncurses,
+    install : sdk_version.version_compare('>=12'),
+    sources : 'latency/latency.c'
+)
+
+# Requires SoftLinking/WeakLinking.h
+# executable(
+#     'login',
+#     c_args = '-DUSE_BSM_AUDIT',
+#     install : true,
+#     sources : [
+#         'login/login.c',
+#         'login/login_audit.c'
+#     ]
+# )
+
+# Requires private libproc APIs
+# executable(
+#     'lskq',
+#     install : true,
+#     sources : 'lskq/lskq.c'
+# )
+
+executable(
+    'lsmp',
+    build_by_default : sdk_version.version_compare('>=12'),
+    install : sdk_version.version_compare('>=12'),
+    sources : [
+        'lsmp/lsmp.c',
+        'lsmp/port_details.c',
+        'lsmp/task_details.c'
+    ]
+)
+
+# Requires kernel headers
+executable(
+    'ltop',
+    install : true,
+    sources : 'ltop/ltop.c'
+)
+
+executable('mean', install : true, sources : 'mean/mean.c')
+
+# Requires kernel headers
+# executable('memory_pressure', install : true, sources : 'memory_pressure/memory_pressure.c')
+
+executable('mkfile', install : true, sources : 'mkfile/mkfile.c')
+
+# Requires stack_logging.h
+# executable('mslutil', install : true, sources : 'mslutil/mslutil.c')
+
+executable('newgrp', install : true, sources : 'newgrp/newgrp.c')
+
+executable('nologin', install : true, sources : 'nologin/nologin.c')
+
+# Requires private IOKit APIs
+# executable('nvram', install : true, sources : 'nvram/nvram.c')
+
+custom_target(
+	'pagesize',
+	capture : true,
+	# Replace hard-coded, impure system paths with the output path in the store.
+	command : [ sed, '-e', 's|PATH=[^;]*|PATH="@out@/bin"|', '@INPUT@' ],
```

Comment:
can this be done in the the postPatch phase? modifying source in the nix files or patches is pretty common but putting it in the meson file seems non-standard.

Reply:
Done.